tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1169,6 +1169,9 @@ def get_placeholder_shapes(argv_input: str, argv_input_shape: str, argv_batch=No
             if '->' not in inp:
                 continue
             shape = placeholder_shapes[inp.split('->')[0]]
+
```

Comment:
please update help for `--input` with a new example: `node_name->False`

Reply:
Agree, updated.